config/locales/en_BE.yml

```diff
@@ -2590,7 +2590,7 @@ en_BE:
           display_as: Display As
       reports:
         table:
-          select_and_search: "Select filters and click on SEARCH to access your data."
+          select_and_search: "Select filters and click on %{option} to access your data."
```

Comment:
We only change `en.yml` in development. The other locales get Translated through Transifex. Changing them in a pull request creates merge conflicts. Can you remove your change from all locales except `en.yml`? You can read more about it at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development.

Reply:
Sure, I can do that. Thanks for the link!